Android/app/src/main/java/com/example/kahye/common/Adapter.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
@Kahyelee I see the name of this file contains your name. Is this the thing that you brought up yesterday when we had 1:1?

Reply:
Yes It is. I tried to change it but we had to do other things first. I'll fix it sooner.